### DISCUSSION TITLE
New dedicated Discord channel

### DISCUSSION THREAD

**Discussion by garronej** [keycloakify/keycloakify#326] [Announcements]
Anyone welcome to join: https://discord.gg/kYFZG7fQmn

@willwill96 @mkreuzmayr

**Comment by tobiashaefermann** [comment]
Hi @garronej, the invite link seems to be expired, could you update it? :)

**Reply by garronej** [reply]
Oh yes indeed, thanks for giving me the heads up: https://discord.gg/kYFZG7fQmn
